viruses/cancelnil/cancelnil_test.go

```diff
@@ -0,0 +1,30 @@
+package cancelnil_test
+
+import (
+	"testing"
+
+	"github.com/gtramontina/ooze/oozetesting"
+	"github.com/gtramontina/ooze/viruses/cancelnil"
+)
+
+func TestCancelNil(t *testing.T) {
+	oozetesting.Run(t, oozetesting.NewScenarios(
+		"Call cancel(nil)",
+		cancelnil.New(),
+		oozetesting.Mutations{
+			// "no mutations": {"source.0.go", []string{}},
```

Comment:
This seems related to https://github.com/gtramontina/ooze/blob/2e027fe94db085b15599cfaa04411c64efb5e2ec/viruses/cancelnil/testdata/source.0.go?

Reply:
Whoops, this shouldn't have been commented out.

Reply:
```suggestion
			"no mutations": {"source.0.go", []string{}},
```